Pod/Tests/TestStash.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+import XCTest
```

Comment:
Agreed :) But like I said, we could use it for all the things if we update the `pod-template`, which might be a good idea :)